source/features/view-last-pr-deployment.tsx

```diff
@@ -0,0 +1,27 @@
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import observeElement from '../helpers/simplified-element-observer';
+
+const deploymentSelector = '.js-timeline-item [data-url$="deployed"] .TimelineItem-body .btn[target="_blank"]';
+
+function init(): void {
+	const lastDeployment = select.last(deploymentSelector)!.cloneNode(true);
+	lastDeployment.classList.add('mr-1');
+	lastDeployment.textContent = 'View last deployment';
+
+	select('.gh-header-actions')!.prepend(lastDeployment);
```

Comment:
I think it's safer to just generate the link via JSX and only copy the URL from the button. It shouldn't be much longer
```suggestion
	const {href} = select.last(deploymentSelector)!;
	select('.gh-header-actions')!.prepend(
		<a className="btn mr-1" href={url} target="_blank">Last deployment</a>
	);
```

Reply:
Maybe if you include `a` in the selector it will use HTMLAnchorElement automatically

Reply:
> I think it's safer

Which part?

Reply:
> Maybe if you include `a` in the selector it will use HTMLAnchorElement automatically

no luck